Mathlib/Analysis/Meromorphic/Divisor.lean

```diff
@@ -116,4 +126,26 @@ theorem divisor_inv [CompleteSpace 𝕜] {f : 𝕜 → 𝕜} :
   · simp [divisor_apply, h, (h.1 z h.2).order_inv]
   · simp [divisor_def, h]
 
+/-- Adding an analytic function to a meromorphic one does not change the pole divisor. -/
+theorem divisor_add_analytic {f₁ f₂ : 𝕜 → E} (hf₁ : MeromorphicOn f₁ U)
+    (hf₂ : AnalyticOnNhd 𝕜 f₂ U) :
+    (divisor f₁ U)⁻ = (divisor (f₁ + f₂) U)⁻ := by
```

Comment:
Can you swap the equality around? RHS is currently more complicated than LHS

Reply:
Done.